201-vmss-ubuntu-web-ssl/azuredeploy.json

```diff
@@ -0,0 +1,435 @@
+      "allowedValues": [
+        "14.04.4-LTS"
+      ],
+      "metadata": {
+        "description": "The Ubuntu version for the VM. This will pick a fully patched image of this given Ubuntu version. Allowed values are: 15.10, 14.04.4-LTS."
+      }
+    },
+    "vmssName": {
+      "type": "string",
+      "metadata": {
+        "description": "String used as a base for naming resources. Must be 3-61 characters in length and globally unique across Azure. A hash is prepended to this string for some resources, and resource-specific information is appended."
+      },
+      "maxLength": 61
+    },
+    "capacity": {
+      "type": "int",
+      "metadata": {
+        "description": "Number of VM instances (100 or less)."
+      },
+      "maxValue": 100
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Admin username on all VMs."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Admin password on all VMs."
+      }
+    },
+    "vaultName": {
+      "type": "string",
+      "metadata": {
+        "description": "The Azure Key vault where SSL certificates are stored"
+      }
+    },
+    "vaultResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "Resource Group of the key vault"
+      }
+    },
+    "httpssecretUrlWithVersion": {
+      "type": "string",
+      "metadata": {
+        "description": "full Key Vault Id to the secret that stores the SSL cert"
+      }
+    },
+    "httpssecretCaUrlWithVersion": {
+      "type": "string",
+      "metadata": {
+        "description": "full Key Vault Id to the secret that stores the CA cert"
+      }
+    },
+    "location": {
+      "type": "string",
+      "metadata": {
+        "description": "location of the deployment"
+        },
+        "allowedValues" : [
+          "eastus",
+          "eastus2",
+          "westus",
+          "northcentralus",
+          "southcentralus",
+          "northeurope",
+          "westeurope",
+          "westcentralus",
+          "westus2"
+        ]
+      },
+    "scriptUrlBase": {
+      "type": "string",
+      "metadata": {
+        "description": "URL to the blob container where the configuration script is stored"
+      }
+    },
+    "scriptFileName": {
+      "type": "string",
+      "metadata": {
+        "description": "the file name of the script configuring the VMs in the scale set"
+      }
+    },
+    "certThumbPrint": {
+      "type": "string",
+      "metadata": {
+        "description": "fingerprint of the SSL cert"
+      }
+    },
+    "caCertThumbPrint": {
+      "type": "string",
+      "metadata": {
+        "description": "fingerprint of the CA cert"
+      }
+    },
+    "scriptStorageResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "resource group for the script's storage account"
+      }
+    },
+    "scriptStorageAccount": {
```

Comment:
could you rename this to _artifactsLocation? (documentation here: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)